cmd/smee/main.go

```diff
@@ -255,6 +258,17 @@ func main() {
 		if err != nil {
 			panic(fmt.Errorf("failed to create backend: %w", err))
 		}
+		// Will print 10% of entries.
+		option := slogsampling.UniformSamplingOption{
+			// The sample rate for sampling traces in the range [0.0, 1.0].
+			Rate: 0.002,
```

Comment:
How does the logging look like at this rate. Do we still get few requests here to give some context that the file is being served or we are dropping too many requests from logs?